WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -751,6 +751,10 @@ - (void)loginSimperium
 
 - (void)logoutSimperiumAndResetNotifications
 {
+    if (self.testSuiteIsRunning) {
```

Comment:
Yes sir! this method will be entirely gone, just a temporary patch.

Thanks for checking!